input/docs/writing-builds/preprocessor-directives/addin.md

```diff
@@ -1,49 +1,128 @@
 Order: 10
 Title: Addin directive
-Description: Directive to install and reference assemblies using NuGet
+Description: Directive to install and reference addins using NuGet
 ---
 
-The add-in directive is used to install and reference assemblies using NuGet.
+Supported in: **[Cake .NET Tool] | [Cake runner for .NET Framework] | [Cake runner for .NET Core]**
 
-# Usage
+The addin directive is used to install and reference addins using NuGet.
 
-The directive takes a single package URI parameter
-Right now, only NuGet packages are supported.
+:::{.alert .alert-info}
+Addin directives is not available when running with [Cake Frosting].
+Instead addins can be added as NuGet packages to the [Cake Frosting] project.
+:::
+
```

Comment:
```suggestion
#addin nuget:?package=Cake.Foo&version=1.0.0
```

Perhaps start with the most common example of package ID and version before expanding on details?